src/UnityExtension/Assets/Editor/GitHub.Unity/UI/Window.cs

```diff
@@ -31,11 +31,17 @@ class Window : BaseWindow
         [SerializeField] private HistoryView historyView = new HistoryView();
         [SerializeField] private SettingsView settingsView = new SettingsView();
 
+        [SerializeField] private string repoRemote;
         [SerializeField] private string repoBranch;
         [SerializeField] private string repoUrl;
         [SerializeField] private GUIContent repoBranchContent;
         [SerializeField] private GUIContent repoUrlContent;
 
+        [SerializeField] private CacheUpdateEvent lastCurrentBranchAndRemoteChangedEvent;
+        [NonSerialized] private bool currentBranchAndRemoteHasUpdate;
+
+        [NonSerialized] private bool hasRunMaybeUpdateDataWithRepository;
```

Comment:
What's this flag for?

Reply:
When `Window` gets it's instance of `Repository` `currentBranchAndRemoteHasUpdate` was still false.
For some reason it's not having the effect I thought it did. It seems i can remove it.